ale_linters/php/phpcs.vim

```diff
@@ -1,15 +1,24 @@
-" Author: jwilliams108 <https://github.com/jwilliams108>
+" Author: jwilliams108 <https://github.com/jwilliams108>, Eric Stern <https://github.com/firehed>
 " Description: phpcs for PHP files
 
 let g:ale_php_phpcs_standard = get(g:, 'ale_php_phpcs_standard', '')
 
+function! ale_linters#php#phpcs#GetExecutable(buffer) abort
+    return ale#path#ResolveLocalPath(
+    \   a:buffer,
+    \   'vendor/bin/phpcs',
+    \   'phpcs'
+    \)
```

Comment:
Could you make the global executable configurable and add a `use_global` option while you're at it, similar to eslint and others?